### PR TITLE
topology2: Add 4ch audio formats for gain-capture pipeline

### PR DESCRIPTION
**Pull request by RanderWang**
Add support in gain-capture pipeline to support 4ch DMIC stream.

### CODE REVIEW

**Comment by aiChaoSONG** [comment]
since you are touching this conf, could you please help to update to use input_audio_format and output_audio_format class, instead of audio_format? audio_format will be deprecated and remove in the future I think.

**Review by lgirdwood — APPROVED** [review]
@ranj063

File: tools/topology/topology2/include/pipelines/cavs/gain-capture.conf [hunk 1]
```diff
@@ -69,12 +69,26 @@ Class.Pipeline."gain-capture" {
 				out_bit_depth		32
 				out_valid_bit_depth	32
 			}
+			# 32-bit 48KHz 4ch
+			Object.Base.audio_format.4 {
```
Comment (ranj063) [inline tools/topology/topology2/include/pipelines/cavs/gain-capture.conf:73]: @RanderWang please use the input_audio_format and output_audio_format objects explicitly

Comment (RanderWang) [reply tools/topology/topology2/include/pipelines/cavs/gain-capture.conf:73]: NP. Thanks!

**Comment by RanderWang** [comment]
updated with input@ouput audo format

**Review by kv2019i — APPROVED** [review]

**Pull request merged**